content/en/docs/setup/_index.md

```diff
@@ -100,7 +100,7 @@ The following production environment solutions table lists the providers and the
 | [Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)          | &#x2714;             |   &#x2714;     |             |      |          |
 | [oVirt](https://www.ovirt.org/)         |              |        |             |     | &#x2714;          |
 | [Pivotal](https://pivotal.io/) | | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service) | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service)           |      |          |
-| [Platform9](https://platform9.com/)         | &#x2714;             | &#x2714;      | &#x2714;            |      |   &#x2714;       |&#x2714;
+| [Platform9](https://platform9.com/)         | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)             |       | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)            |  [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)    |   [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)       |[Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)
```

Comment:
```suggestion
| [Platform9](https://platform9.com/)         | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)             |       | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)            |  [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)    |   [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)    |   &#x2714;
```

Reply:
Hi @jaypipes I hope this is what you meant. Basically, we need to include direct links to the product page itself since our website (linked on the left column, from the company name) includes other additional products (Managed OpenStack, for example). but I pared down the 'custom' categories to checkmarks. Hope this works?